samples/python/videocapture_obsensor.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import numpy as np
+import sys
+import cv2 as cv
+
+
+def main():
+    # Open Orbbec depth sensor
+    orbbec_cap = cv.VideoCapture(0, cv.CAP_OBSENSOR)
+    if orbbec_cap.isOpened() == False:
+        sys.exit("Fail to open camera.")
+
+    while True:
+        # Grab data from the camera
+        if orbbec_cap.grab():
+            # RGB data
+            ret_bgr, bgr_image = orbbec_cap.retrieve(None, cv.CAP_OBSENSOR_BGR_IMAGE)
+            if ret_bgr:
+                cv.imshow("BGR", bgr_image)
+
+            # depth data
+            ret_depth, depth_map = orbbec_cap.retrieve(None, cv.CAP_OBSENSOR_DEPTH_MAP)
+            if ret_depth:
+                norm_depth_map = cv.normalize(depth_map, None, 0, 255, cv.NORM_MINMAX, cv.CV_8UC1)
```

Comment:
How about adding code to convert the depth to a color map?
Since we have a cpp sample at https://github.com/opencv/opencv/blob/4.x/samples/cpp/videocapture_obsensor.cpp#L40, I think we better have a unified output.

Reply:
Done.